app/src/main/java/org/ieeeguc/ieeeguc/models/Meeting.java

```diff
@@ -173,5 +178,77 @@ public Attendee(User user, String review, int rating) {
             this.rating = rating;
         }
     }
+
+    /**
+     * @param  {string} token  (token of the requesting user )
+     * @param  {int} id (the id of the meeting)
+     * @param {int} rating (the rating of the meting)
+     * @param {boolean Array} goals (boolean array that states if the goals of the meeting have been achived or not)
+     * @param {JSONArray} attendees (JSOnArray that have the ratings and review of every member of the meeting)
+     * @param {HttpResponse} HTTP_RESPONSE (http interface instance which is the response coming from the server after logging in containing info of the user in the Database)
+     */
+    public void rate(String token,int id, int rating,  boolean [] goals  , JSONArray attendees,final HTTPResponse HTTP_RESPONSE){
+        OkHttpClient client= new OkHttpClient();
+        JSONObject jsonBody = new JSONObject();
+        try {
+            jsonBody.put("evaluation",rating);
+            jsonBody.put("goals",goals);
+            jsonBody.put("attendees",attendees);
+            String url = "http://ieeeguc.org/api/meeting/"+ id +"/rate";
+            RequestBody body = RequestBody.create(MediaType.parse("application/json; charset=utf-8"), jsonBody.toString());
+            Request request=new Request.Builder()
+                    .url("http://ieeeguc.org/api/User")
+                    .addHeader("Authorization",token)
+                    .addHeader("user_agent","Android")
+                    .post(body)
+                    .build();
+            client.newCall(request).enqueue(new Callback() {
+                public void onFailure(Call call, IOException e) {
+                    MainActivity.UIHandler.post(new Runnable() {
+                        @Override
+                        public void run() {
+                            HTTP_RESPONSE.onFailure(-1,null);
+                        }
+                    });
+                    call.cancel();
+                }
+                public void onResponse(Call call, okhttp3.Response response)  {
+                    try {
+                        String body = response.body().string();
+                        final JSONObject bodyJSON = new JSONObject(body);
+                        final int statusCode = response.code();
+                        MainActivity.UIHandler.post(new Runnable() {
```

Comment:
before going to the controller update the current meeting with the new data you just posted.